.github/workflows/build_test_push.yaml

```diff
@@ -54,6 +54,8 @@ jobs:
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@master
+        with:
```

Comment:
This was needed due to a weird detail of how `load: true` works. Some context [here](https://github.com/docker/build-push-action/issues/321), it failed without this and then worked when I added it, so I didn't dig too much deeper 🤷